tools/ci_build/github/android/nnapi_supported_ops.md

```diff
@@ -0,0 +1,48 @@
+<!-- 
+Keep in sync with doco generated from /docs/execution-providers/NNAPI-ExecutionProvider.md on the gh_pages branch
+-->
```

Comment:
No easy way to handle the docs being generated from a different branch. I added parsing of the md in the scripts so it's simply cut-and-paste between the two locations to keep things in sync.

Reply:
how will this syncing work with release branches?

Reply:
The md file gets copied into the python package, so the supported ops information used by the tool would be specific for that release.

If they don't match the python ORT version with the pre-built package version I can't do much about that. In theory we could maintain diffs and ask them to specify a version but that's a lot of work for very little benefit.